codec-http/src/main/java/io/netty/handler/codec/http/HttpMethod.java

```diff
@@ -86,7 +87,7 @@
      */
     public static final HttpMethod CONNECT = new HttpMethod("CONNECT");
 
-    private static final Map<String, HttpMethod> methodMap = new HashMap<String, HttpMethod>();
+    private static final Map<String, HttpMethod> methodMap = new TreeMap<String, HttpMethod>();
```

Comment:
this was on the hot path during benchmarks ... I may pull this out into another PR ... or at the very least merge it back to 4.0.